docusaurus/sidebars-ios.json

```diff
@@ -19,6 +19,7 @@
         },
         "uikit/components/channel",
         "uikit/views/channel-header-view",
+        "uikit/components/thread-list",
```

Comment:
This is too high in the sidebar - it should go after the main components, probably after the composer.

Reply:
My reason would be to move it closer to the Channel List, since most likely they are displayed close to each other 🤔

Reply:
This is a bit of a niche component, so it should be a bit lower in the hierarchy - we want people to first see the message list and the composer.

Reply:
Done ✅